docs/webhooks.rst

```diff
@@ -26,10 +29,14 @@ project was not imported through a connected account, you may need to
 manually configure a webhook for your project.
 
 To manually set up a webhook, click **Add integration** on your project's
-**Integrations** admin dashboard page and select the integration type you'd like
+**Integrations** Admin dashboard page and select the integration type you'd like
 to add. After you have added the integration, you'll see a URL for the
-integration on the :ref:`integration detail page <webhooks:Webhook Integrations>`. Use this
-URL when setting up a new webhook with your provider -- these steps vary
+integration once you go to readthedocs.org/dashboard, log in, and then go to 
+**Admin** > **Integrations** to see or create an integration.
+
+As an example, the URL pattern looks like this: *readthedocs.org/api/v2/webhook/<project-name>/<nnnnnn>/*.
```

Comment:
We can use `<id>` instead of `<nnnn>`

Reply:
Thanks - and is that formatting okay with angle brackets and italic?

Reply:
Yeah, I think is fine